docs/DafnyRef/Types.md

```diff
@@ -1,32 +1,18 @@
-# 6. Types {#sec-types}
-````grammar
-Type = DomainType_ | ArrowType_
-````
+# 5. Types {#sec-types}
+
 A Dafny type is a domain type (i.e., a type that can be the domain of an
```

Comment:
Outside the scope of this PR, but I find this a curious definition. Why don't you consider arrow types just another kind of type, instead of saying types are either domain types or arrow types consisting of a domain and a range type?

Reply:
Probably Rustan's (or maybe Ford's) words originally, but I'll think about a revision.

Reply:
Revised.